src/main/java/com/fouristhenumber/utilitiesinexcess/common/recipe/RecipeLoader.java

```diff
@@ -12,35 +12,28 @@ public class RecipeLoader {
 
     public static void run() {
         if (Mods.Dreamcraft.isLoaded()) return;
-        loadCompressedCobblestoneRecipes();
+        loadCompressedBlockRecipes();
     }
 
-    private static void loadCompressedCobblestoneRecipes() {
-
-        GameRegistry.addRecipe(
-            new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 1, 0),
-            "###",
-            "###",
-            "###",
-            '#',
-            new ItemStack(Blocks.cobblestone, 9, 0));
-
-        GameRegistry.addShapelessRecipe(
-            new ItemStack(Blocks.cobblestone, 9, 0),
-            new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 1, 0));
-
-        for (int i = 0; i < 7; i++) {
-            GameRegistry.addRecipe(
-                new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 1, i + 1),
-                "###",
-                "###",
-                "###",
-                '#',
-                new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 1, i));
-
-            GameRegistry.addShapelessRecipe(
-                new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 9, i),
-                new ItemStack(ModBlocks.COMPRESSED_COBBLESTONE.get(), 1, i + 1));
+    private static void loadCompressedBlockRecipes() {
+
+        ModBlocks[] blocks = { ModBlocks.COMPRESSED_COBBLESTONE, ModBlocks.COMPRESSED_DIRT, ModBlocks.COMPRESSED_GRAVEL,
+            ModBlocks.COMPRESSED_SAND, };
+
+        for (ModBlocks modBlock : blocks) {
+            if (!(modBlock.get() instanceof BlockCompressed block) || !modBlock.isEnabled()) return;
```

Comment:
```suggestion
            if (!(modBlock.get() instanceof BlockCompressed block) || !modBlock.isEnabled()) continue;
```
Should skip this block but continue with the rest

Reply:
Good catch